### PR TITLE
github-autobuild: change to build once a day

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

**Comment by evan-charmworks** [comment]
Should this be at 01:00 to match current autobuild?

**Comment by matthiasdiener** [comment]
> Should this be at 01:00 to match current autobuild?

Not sure. We might want to run it at a slightly different time in order not to overlap with the current autobuild.

File: .github/workflows/autobuild.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Autobuild
 
 on:
   schedule:
-    - cron: '59 * * * *'
+    - cron: '01 05 * * *' # UTC, corresponds to 00:01 Central Time
```
Comment (rbuch) [inline .github/workflows/autobuild.yml:5]: To be pedantic, if this is UTC, then it should vary between 23:01 and 00:01 (CST and CDT, respectively).

Comment (matthiasdiener) [reply .github/workflows/autobuild.yml]: good point, fixed.

**Pull request merged**